package.json

```diff
@@ -9,14 +9,21 @@
     "packages/db"
   ],
   "scripts": {
+    "prepare": "husky",
     "lint": "eslint --ext .ts,.tsx .",
+    "lint:fix": "eslint --fix --ext .ts,.tsx .",
+    "typecheck": "npm run --workspaces typecheck",
```

Comment:
Runs the typecheck command from each nested package (api, db, fe). This command has been added to each of those package.json's